docs/context/ref/ChatInitSettings.md

```diff
@@ -0,0 +1,85 @@
+---
+id: ChatInitSettings
+sidebar_label: ChatInitSettings
+title: ChatInitSettings
+hide_title: true
+---
+
+# `ChatInitSettings`
+
+A collection of settings to start a new chat conversation
+
+## Type
+
+`fdc3.chat.initSettings`
+
+## Schema
+
+https://fdc3.finos.org/schemas/next/chatInitSettings.schema.json
+
+## Details
+
+| Property                       | Type      | Required | Example Value                                                        |
+| ------------------------------ | --------- | -------- | -------------------------------------------------------------------- |
+| `type`                         | string    | Yes      | `'fdc3.chat.initSettings'`                                           |
+| `chatName`                     | string    | No       | `'Instrumet XYZ'`                                                    |
+| `members`                      | Contact[] | No       | `[contact1, contact2]`                                               |
```

Comment:
Maybe we could use [`ContactList`](https://fdc3.finos.org/docs/context/ref/ContactList) instead of `Contact[]`.

Reply:
Yes - I though of this option, but I realised that ContactList adds extra verbosity (type, sub-structure, etc.) this is why I preferred the Contact[] option

Reply:
OK - I will update the PR with Contact and ContactList then